src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -327,5 +327,13 @@ public async Task<ActionResult> WhenDoYouWantToView(int? choice, string agreemen
                 }
             }
         }
+
+        [HttpGet]
+        [Route("agreements/{accountLegalEntityHashedId}/viewallagreements")]
```

Comment:
This doesn't fit in with all the other methods which follow this pattern

accounts/{HashedAccountId}/agreements or 
accounts/{HashedAccountId}/agreements/{agreementId}

Just for info how does accountLegalEntityHashedId fit in with this?  

Should it be?

accounts/{HashedAccountId}/organisations/{accountLegalEntityHashedId}/agreements

Reply:
This is been done as part of CON-1492 :  i have inherited , so that testers can test this as stand alone

Reply:
we should address when either branch has gone to master and been merged back up

Reply:
done

Reply:
I don't understand the comment.  I think this is incorrect.  Please run past Corey.

Reply:
I checked with Corey, initially  thought part of CON-1492 , then its been removed later.
hence Corey replied : i am able to make the change. Changes been done.